include/Annotation.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <string>
+
+#include "Rotation.hpp"
+#include "Translation.hpp"
+#include "nlohmann/json.hpp"
+
+using json = nlohmann::json;
+
+class Annotation {
+ private:
+  std::string token;
+  std::string sample_token;
+  std::string instance_token;
+  std::string visibility_token;
+  std::vector<std::string> attribute_tokens;
+  Translation translation;
+  Rotation rotation;
+  std::array<double, 3> size;
+  std::string prev;
+  std::string next;
+
+ public:
+  const std::string &get_token() const;
+  const std::string &get_sample_token() const;
+  const std::string &get_instance_token() const;
+  const std::string &get_visibility_token() const;
+  const std::vector<std::string> &get_attribute_tokens() const;
+  const Translation &get_translation() const;
+  const Rotation &get_rotation() const;
+  const std::array<double, 3> &get_size() const;
+  const std::string &get_prev() const;
+  const std::string &get_next() const;
+  Annotation() = delete;
+  Annotation(const json &);
+  Annotation(std::string, std::string, std::string, std::string,
```

Comment:
Passing `std::string` to a constructor by value induces unnecessary copy for initialization member data.

Reply:
Thanks, I'll fix it and avoid unnecessary copy it in the next PR.

Reply:
I've merged it in #7